.gitignore

```diff
@@ -24,3 +24,7 @@ Thumbs.db
 UserInterfaceState.xcuserstate
 .env
 src/**/components.d.ts
+
+# backup files
+*.bak
```

Comment:
Where are these files coming from? If not from an IDE, I'd suggest using git stash or local branches.

Reply:
> I'd suggest using git stash or local branches.
👍 
 I don't feel strongly about keeping it it gitignore -  but its a small benefit to keep these file types ignored. Are you also asking to remove this?

Reply:
Not right now. I think in the future this file should be revisited.